specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,182 @@
+                                args2;
+
+                            args->QueryInterface(IID_PPV_ARGS(&args2));
+                            if (args) 
+                            {
+                                if (key == VK_F7)
+                                {
+                                    // Allow the browser to process F7 key
+                                    CHECK_FAILURE(args2->put_AllowBrowserAcceleratorHandling(TRUE));
+                                }
+                            }
+                        }
+                        return S_OK;
+                    })
+                    .Get(),
+                &m_acceleratorKeyPressedToken));
+
+        }
+    }
+```
+
+## C#
+```c#
+    CoreWebView2Settings _webViewSettings = webView.CoreWebView2.Settings;
+    // All browser accelerator keys are disabled with this setting. The default value for the 
+    // `AllowBrowserAcceleratorHandling`property is set to the current value of this setting. 
+    // Changing this setting to `false` will change the default setting for 
+    // AllowBrowserAcceleratorHandling to `false`
+    _webViewSettings.AreBrowserAcceleratorKeysEnabled = false;
+
+    webView.CoreWebView2.AcceleratorKeyPressed += WebView2Controller_AcceleratorKeyPressed;
+
+    void WebView2Controller_AcceleratorKeyPressed(object sender, CoreWebView2AcceleratorKeyPressedEventArgs e)
+    {
+        switch (e.KeyEventKind)
+        {
+            case CoreWebView2KeyEventKind.KeyDown:
+            case CoreWebView2KeyEventKind.SystemKeyDown:
+                {
+                    // Allow the browser to process F7 key
+                    if (e.VirtualKey == Key.F7) 
+                    {
+                        e.AllowBrowserAcceleratorHandling = true;
+                    }
+                    break;
+                }
+            
+        }
+    }
+
+```
+
+# Remarks
+By default, `AreBrowserAcceleratorKeysEnabled` is `TRUE` and default value for `AllowBrowserAcceleratorHandling`
+is `TRUE`. When developers set `AreBrowserAcceleratorKeysEnabled` to `FALSE` this will also change the default 
+value for `AllowBrowserAcceleratorHandling` to `FALSE`. If developers want specific keys to be handled by the 
+browser after changing the `AreBrowserAcceleratorKeysEnabled` settings to `FALSE`, they need to enable them by 
+setting `event_args->put_ AllowBrowserAcceleratorHandling(TRUE)`. This option will give the event arg higher 
+priority over the settings when the browser decide whether to handle the keys.  
+
+# API Details
+## C++
+```
+/// This is a continuation of the ICoreWebView2AcceleratorKeyPressedEventArgs interface.
+[uuid(45238725-3774-4cfe-931f-7985a1b5866f), object, pointer_default(unique)]
+interface ICoreWebView2AcceleratorKeyPressedEventArgs2 : ICoreWebView2AcceleratorKeyPressedEventArgs {
+  /// Browser accelerator keys are the keys/key combinations that access features specific to 

```

Comment:
You need the first sentence to be a summary of the property. Something like the following maybe

```suggestion
  /// This property allow you to enable or disable the browser from handling a specific browser 
  /// accelerator key such as Ctrl+P or F3.
  /// Browser accelerator keys are the keys/key combinations that access features specific to 
```